### ISSUE TITLE
XML documentation bug in DefaultValidatorExtensions.IsEnumName<T>()

### ISSUE THREAD

**Issue by jehrenzweig** [JeremySkinner/FluentValidation#1241]
The [IsEnumName<T>()](https://github.com/JeremySkinner/FluentValidation/blob/1897f226c0aa07ddc3cb88bf05026122a52cca54/src/FluentValidation/DefaultValidatorExtensions.cs#L1099) extension method doesn't have a `TProperty` type parameter, even though the xmldocs mention one:
```
/// <typeparam name="TProperty">Type of property being validated</typeparam>
```

I suspect a copy of `IsInEnum<T, TProperty>()` was used as a starting point when creating `IsEnumName<T>()` and the last line of the xmldocs were missed when updating the documentation.

**Comment by JeremySkinner** [comment]
Thanks, I suspect it was copy and pasted when the original developer contributed it. Feel free to submit a PR, otherwise I’ll correct it when I get some time.

**Comment by jehrenzweig** [comment]
I'll carve out a PR and link it here once it's up -- I just wanted to confirm it before I did anything.  Danke!

**Comment by JeremySkinner** [comment]
Fixed in 7289d60
